Source/WebKit/UIProcess/API/glib/WebKitWebView.cpp

```diff
@@ -2257,15 +2257,20 @@ static void webkit_web_view_class_init(WebKitWebViewClass* webViewClass)
      *
      * Emitted after #WebKitWebView::context-menu signal, if the context menu is shown,
      * to notify that the context menu is dismissed.
+     *
+     * Deprecated: 2.48, WPE WebKit does not emit this signal.
```

Comment:
This is deprecating the signal for GTK too. We need to split the documentation and definition like the context-menu signal.